pytest-selenium/tests/test_navigation.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from pages.content import Content
+from tests import markers
+
+
+@markers.test_case("C477321")
+@markers.parametrize("page_slug", [("preface")])
+@markers.nondestructive
+def test_previous_link_hidden_on_first_page(selenium, base_url, book_slug, page_slug):
+
+    # GIVEN: The page is loaded
+    content = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
+    toolbar = content.toolbar
+    toc = content.sidebar.toc
+
+    # confirm first page is selected
+    if toc.active_section.get_attribute("textContent") != toc.sections[0].section_title:
+        while content.is_mobile:
+            toolbar.click_toc_toggle_button()
+            break
+        toc.sections[0].click()
```

Comment:
would it be better to reorganize these helpers to look more like

```
if not toc.first_page.is_active:
  if content.is_mobile:
    toolbar.click_toc_toggle_button()
  toc.first_page.click()
```

Reply:
@TomWoodward reorganized the helpers like you mentioned and the test really looks short :-)